it/testkit/CommonHttpWorkflows.scala

```diff
@@ -49,9 +49,7 @@ trait CommonHttpWorkflows { self: WireLevelHttpSpec =>
            |   },
            |  "postCode":"H0H 0H0",
            |  "mobilePhone":"071234567",
-           |  "recentSchool":"Super hero's academy",
-           |  "aLevel": $aLevelsAboveD,
-           |  "stemLevel": $aLevelsAboveCInStem
+           |  "recentSchool":"Super hero's academy"
```

Comment:
Overly pedantic grammar police notice: Did you mean "Superheroes' academy", "Superhero's academy" or "Superheroes academy"